src/atoms/MenuItem/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled, { css } from '../../lib/styledComponents';
+
+interface Props {
+  children: string | React.ReactNode;
+  active?: boolean;
+  onClick?: () => any;
+  'data-qaid'?: string;
+}
+
+export default styled.li<Props>`
+  ${({ theme, active }) => css`
+    margin-top: 0px;
+    list-style: none;
+    padding: ${theme.ruler[2]}px ${theme.ruler[4]}px;
+    font-size: ${theme.fontSizes.body2};
+    letter-spacing: 0.25px;
```

Comment:
Do we have to use this here? Do we need a new definition perhaps for body text that used in 'functional' locations?

Reply:
Yeah it doesn't _really_ match the Body or Body2 styling we have. Not a big deal I think. But can change if you want! I think, Body is `0.3px` letterSpacing 🤷‍♂